software/controller/lib/core/system_constants.h

```diff
@@ -23,10 +23,32 @@ limitations under the License.
 #include "gpio.h"
 #include "units.h"
 
+/**************************************************************************************
+ *                                PROCESSOR CONSTANTS                                 *
+ **************************************************************************************/
+
+/// \TODO ensure CPUFrequency is a multiple of 10 MHz
+static constexpr Frequency CPUFrequency{megahertz(80)};
+static constexpr Frequency UARTBaudRate{hertz(115200)};
+
 /**************************************************************************************
  *                                ELECTRICAL CONSTANTS                                *
  **************************************************************************************/
 
+/******************************************************************
+ * PWM outputs
+ *
+ * The following four outputs could be driven
+ * as PWM outputs:
+ *
+ * PA8  - Timer 1 Channel 1 - heater control
+ * PA11 - Timer 1 Channel 4 - psol control
+ * PB3  - Timer 2 Channel 2 - blower control
+ * PB4  - Timer 3 Channel 1 - buzzer control
+ *
+ * These timers are documented in [RM] chapters 26 and 27.
+ *****************************************************************/
+
```

Comment:
Mappings are redundant with the individual comments on each PwmChannel, and actually the documentation in [RM] is related to the timers, and belongs in the pwm implementation which sets the timer registers.

Reply:
ok